<?xml version="1.0"?>

<launch>
  <arg name="detected_persons_topic" default="detected_persons"/>
  <arg name="object_detection_topic" default="object_detection"/>
  
  <node pkg="tuw_object_converter" type="detection_to_spencer_detected_person_node" name="detection_to_spencer_detected_person_node_$(arg object_detection_topic)" output="screen">
    <remap from="detected_persons_topic" to="$(arg detected_persons_topic)" />
    <remap from="object_detection_topic" to="$(arg object_detection_topic)" />
  </node>
</launch>

